core/export.ts

```diff
@@ -0,0 +1,158 @@
+import { loadValue } from '@logux/client'
+import { atom, onMount } from 'nanostores'
+
+import {
+  type CategoryValue,
+  feedsByCategory,
+  getCategories,
+  getCategoryTitle
+} from './category.js'
+import { type FeedValue, getFeeds } from './feed.js'
+import { readonlyExport } from './lib/stores.js'
+import { getPosts, type PostValue } from './post.js'
+
+type ExtendedFeedValue = {
+  posts?: PostValue[]
+} & FeedValue
+
+let $categories = atom<CategoryValue[]>([])
+let $allFeeds = atom<FeedValue[]>([])
+let $feedsByCategoryList = atom<[CategoryValue, ExtendedFeedValue[]][]>([])
+let $creating = atom<boolean>(false)
+
+export const feedsByCategoryList = readonlyExport($feedsByCategoryList)
+export const creating = readonlyExport($creating)
```

Comment:
Let’s add `export` word to all exports. Otherwise `creating` will have name conflict with other files in the future.